game/util.hh

```diff
@@ -5,6 +5,8 @@
 #include <vector>
 #include <stdexcept>
 
+constexpr double m_pi = 3.141592653589793238462643383279502884;
```

Comment:
Please define pi somewhere where all classes and namespaces can use it.
Now the code is 'cluttered' with 2 random pi digits. Effected place here and `game/libda/fft.hpp#L15` als i wouldn't make it a fixed digit i'd calculate it like the following: `constexpr double pi() { return std::atan(1)*4; }`
`std::atan()` is defined in `#include <cmath>`

Reply:
It is recommended to use plain digits because math functions might be slightly inaccurate for performance reasons. Also, atan is not constexpr so it cannot be evaluated at compile time (by c++ standard).

As for da::TAU - I want to keep libda a stand-alone library (that can be used by projects other than Performous). Also, including libda headers in parts of Performous that don't use libda doesn't seem like a good idea, so this why Performous uses the definition from util.hh.

I think that m_pi should be renamed to something that doesn't look like a member variable. Did you try "PI" already, or is that name already taken by other libraries? I suppose that we could also in Performous only define TAU, because most of the time we really want 2pi. See https://tauday.com/tau-manifesto